app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/main/MovieApplication.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.marcinwisniewski.intivetrainingapp.main
+
+import android.app.Application
+import android.support.v4.app.Fragment
+import com.example.marcinwisniewski.intivetrainingapp.dagger.component.DaggerAppComponent
+import dagger.android.AndroidInjector
+import dagger.android.DispatchingAndroidInjector
+import dagger.android.support.HasSupportFragmentInjector
+import javax.inject.Inject
+
```

Comment:
Standard application class extension - it implements support fragment injector in case of possibility to do the inject in support fragments